runtime/src/fips.rs

```diff
@@ -130,6 +130,8 @@ pub mod fips_self_test_cmd {
     pub(crate) fn execute(env: &mut Drivers) -> CaliptraResult<()> {
         caliptra_drivers::report_boot_status(RtFipSelfTestStarted.into());
         cprintln!("[rt] FIPS self test");
+        cprintln!("[kat] SHA2-256");
+        caliptra_kat::Sha256Kat::default().execute(&mut env.sha256)?;
```

Comment:
IMO, this looks fine. Just to be clear, the reason this is needed is because the SHA256 KAT needs to be run before performing the ROM integrity test (since SHA256 is used for that test). Is that right?

Reply:
That's correct. Just like the ROM we need the SHA256 KAT to run before we use it for the ROM integrity test.